worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java

```diff
@@ -61,8 +61,29 @@
      * Get the number of blocks in the region.
      *
      * @return number of blocks
+     * @deprecated use {@link Region#getVolume()} to prevent overflows
      */
-    int getArea();
+    @Deprecated
+    default int getArea() {
+        return (int) getVolume();
+    }
+
+    /**
+     * Get the number of blocks in the region.
+     *
+     * @return number of blocks
+     */
+    default long getVolume() {
+        // TODO Remove default status when getArea is removed.
+        try {
+            if (getClass().getMethod("getArea").getDeclaringClass().equals(this.getClass())) {
```

Comment:
This check is wrong:
```suggestion
            if (getClass().getMethod("getArea").getDeclaringClass().equals(Region.getClass())) {
```